src/main/java/com/mateacademy/entity/PersonDao.java

```diff
@@ -0,0 +1,21 @@
+package com.mateacademy.entity;
+
+import java.util.List;
+
+/** This interface contains methods which we have
+ * to realize for CRUD operations.
+ */
+
+public interface PersonDao {
+
+    void addPerson(Person person);
+
+    void editPerson(Person person, int personId);
+
+    void deletePerson(int personId);
+
+    Person find(int personId);
```

Comment:
better return `Optional<Person>`

Reply:
To the previous comment, why should i add controller?
I don't even have web page for this application